resources/migrations/001_update_migrations.yaml

```diff
@@ -5598,6 +5598,18 @@ databaseChangeLog:
             sql: >-
               DROP VIEW IF EXISTS v_content;
 
+  - changeSet:
+      id: v50.2024-03-02T17:41:00
+      author: qnkhuat
+      comment: Add metabase_table.properties
```

Comment:
Remind me, what's the benefit of stuffing this data in a JSON blob over nullable columns? I've tried to make my own list here and I'm not sure it stacks up:

The benefits:
- you can separate fields that are generally required from fields that are specific to certain driver features?
- you don't have to update as many tests

The downsides:
- harder to migrate
- harder to inspect with SQL
- slower to filter/select on

I genuinely feel like I'm missing something though.

Reply:
Agree with Cal. I think this just belongs in a regular column. The old version was a regular column

Reply:
it's now in its own column `metabase_table.estimated_row_count`